packages/dependency-operator-filter/test/fixtures/testFixture.ts

```diff
@@ -124,5 +138,6 @@ export async function setupOperatorFilter() {
     operatorFilterRegistryAsDeployer,
```

Comment:
Remove operatorFilterAsDeployer
No tests should require any deployer action since deployer should not retain any permissions - this means setting up the correct admins eg DEFAULT_ADMIN_ROLE